apps/src/feedback.js

```diff
@@ -987,6 +989,15 @@ FeedbackUtils.prototype.showSimpleDialog = function (Dialog, options) {
   if (options.bodyText) {
     contentDiv.innerHTML += '<p>' + options.bodyText + '</p>';
   }
+  var textBox;
+  if (options.prompt) {
+    textBox = document.createElement('input');
+    textBox.className = "name-textbox";
+    if (options.promptPrefill) {
+      textBox.value = options.promptPrefill;
+    }
+    contentDiv.appendChild(textBox);
```

Comment:
This `contentDiv` could probably be pretty easily replaced by some simple JSX; something like

```jsx
var contentDiv = (
  <div>
    {options.headerText && <p>{options.headerText}</p>}
    {options.bodyText && <p>{options.bodyText}</p>}
    {options.prompt && <input value={options.promptPrefill} />}
  </div>
)
```